unidist/core/backends/mpi/core/serialization.py

```diff
@@ -95,12 +95,12 @@ def _buffer_callback(self, pickle_buffer):
 
     def _dataframe_encode(self, frame):
         """
-        Encode ``pandas.DataFrame`` with pickle library using protocol 5.
+        Encode with pickle library using protocol 5.
 
         Parameters
         ----------
-        frame : pandas.DataFrame
-            Pandas ``DataFrame`` object.
+        frame : object
```

Comment:
```suggestion
        data : object
```
Since we now support any object exposing `__reduce_ex__` method I would rename it to `data.`